.builds/linux.yml

```diff
@@ -6,10 +6,12 @@ tasks:
       curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal -c clippy
   - stable: |
       cd vte
-      $HOME/.cargo/bin/cargo +stable test
+      $HOME/.cargo/bin/cargo +stable test --features=ansi
+      $HOME/.cargo/bin/cargo +stable test --features=ansi,alloc
   - clippy: |
       cd vte
-      $HOME/.cargo/bin/cargo +stable clippy
+      $HOME/.cargo/bin/cargo +stable clippy --features=ansi
+      $HOME/.cargo/bin/cargo +stable clippy --features=ansi,alloc
```

Comment:
Would just `--all-features` work here? I think just having a general test case for all features is probably the most useful testcase. After that a 'minimal' nostd thing could be considered.

Reply:
No, it won't. `--all-features` will enable `nightly` feature, which does not work on stable channel.